### PR TITLE
Fix links in emails

### PR DESCRIPTION
**Pull request by githinho**

### CODE REVIEW

**Review by windhustler — APPROVED** [review]

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -36,15 +36,15 @@ spring.rabbitmq.listener.simple.default-requeue-rejected=true
 # Custom properties
 com.ampnet.mailservice.mail.sender=no-reply@ampnet.io
 com.ampnet.mailservice.mail.base-url=http://localhost
+com.ampnet.mailservice.mail.enabled=false
```
Comment (mightymatth) [inline src/main/resources/application.properties:39]: image

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -36,15 +36,15 @@ spring.rabbitmq.listener.simple.default-requeue-rejected=true
...
-com.ampnet.mailservice.mail.enabled=false
-com.ampnet.mailservice.mail.new-wallet-path=dash/activation
-com.ampnet.mailservice.mail.manage-withdrawals-path=dash/manage_withdrawals
+com.ampnet.mailservice.mail.manage-organization-path=dash/groups
+com.ampnet.mailservice.mail.new-wallet-path=dash/admin/activation
+com.ampnet.mailservice.mail.manage-withdrawals-path=dash/admin/manage_withdrawals
 com.ampnet.mailservice.mail.wallet-activated-path=dash/wallet
-com.ampnet.mailservice.mail.manage-project-path=manage_project
-com.ampnet.mailservice.mail.overview-path=overview
+com.ampnet.mailservice.mail.manage-project-path=dash/project
```
Comment (mightymatth) [inline src/main/resources/application.properties:46]: This is suspicious. Why `project` when the "manage project" screen is here: `https://staging.ampnet.io/ampnet/dash/projects/5aeda05e-1ba4-4be9-8601-deb89cf9cc24/edit`?

Comment (githinho) [reply src/main/resources/application.properties]: It is renamed to `project-path`, and it is used like this: `dash/project` + `/{projectUuid}`. 
There is no needed to force the user to edit screen.

Comment (mightymatth) [reply src/main/resources/application.properties]: `dash/project` route doesn't exist. `dash/projects` and `dash/projects/:uuid` do. (singular -> plural) is the issue.

Comment (githinho) [reply src/main/resources/application.properties]: What is the difference between `dash/projects/:uuid` and `dash/offers/:uuid`?

Comment (mightymatth) [reply src/main/resources/application.properties]: `dash/projects/` shows all the projects (and groups) that can be managed from a project owner's perspective. (suited for project owners and group owners/managers).

`dash/offers` shows the list of all active projects that are available to invest in (suited for investors).

`dash/offers/:uuid` and `dash/projects/:uuid` are **mirrored** and show the project page for investors. The mirroring reason has a purpose. We wanted to split responsibility for investors (offers) and project maintainers (projects). 

I wanted to make it look better from the project manager perspective:

Bad navigation (pay attention to `offers` segment):
- Projects screen: `dash/projects`
- Project screen: `dash/offers/:uuid`
- Project edit screen `dash/projects/:uuid/edit`

Better navigation:
- Projects screen: `dash/projects`
- Project screen: `dash/projects/:uuid`
- Project edit screen `dash/projects/:uuid/edit`

It depends on perspective which route is semantically better.

In your case of managing a project, it is better to route to `dash/projects` or `dash/projects/:uuid`
Does it help?

Comment (githinho) [reply src/main/resources/application.properties]: Awesome! I will use: `dash/projects` + `/:uuid`.

**Pull request merged**